### ISSUE TITLE
The change log is not updated

### ISSUE THREAD

**Issue by Roay** [nfs-ganesha/nfs-ganesha#614]
Why is there no change log for each update？
Version management is maddening！

**Comment by dang** [comment]
New major releases generally have a summary of the major changes.  Minor releases would all have the same changelog: bugfixes.  This is because we don't backport features to stable releases.  The exact bug fixes can be found in the git history (which can be seen on github, if you don't have git available).

**Comment by ffilz** [comment]
I gave up on the change log file(s) (we used to have multiple files to update) when I kept forgetting to update it before tagging a release. We do try and get some change log into the release tag (but I CAN miss that (or in the case of V2.6.0 I messed up the tag, so the actual V2.6.0 tag doesn't have the change log, and in fact that is the only release tag starting with V2.0.0 that doesn't have a change log).

Also, each weekly merge tag has a weekly change log, though it is usually just the list of commits (sometimes I summarize or change wording, other times I don't take the time).

It would be nice if there was an automated process to take the change log from the tag and put that in the RPMs I suppose, but not something we're likely to do at this time.

Closing as won't do any further work on.
